repository/y.json

```diff
@@ -77,6 +77,17 @@
 				}
 			]
 		},
+		{
+			"name": "Yara L2 Syntax",
+			"details": "https://github.com/appliedsciencegroup/sublimetext-package-yaral2-syntax-highlighting",
+			"labels": ["Yara", "Yaral" ,"Yara L2", "syntax", "snippets"],
```

Comment:
`"syntax"` -> `"language syntax"` is the canonical label.

Reply:
I changed syntax label to language syntax.
Thanks